packages/compass-editor/src/json-editor.tsx

```diff
@@ -1148,6 +1148,12 @@ const multilineEditorContainerStyle = css({
   },
 });
 
+const multilineEditorContainerWithActionsStyle = css({
+  minHeight: spacing[5] - 2,
+  display: 'flex',
+  alignItems: 'center',
```

Comment:
BaseEditor is the only element inside this container that is relatively positioned, doesn't look like these styles should be needed 🤔 Can you clarify a bit what are they doing?

Reply:
Yea that's right. Since, BaseEditor is only of `{minLines * lineHeight}px` height, in collapsed state it leaves noticeable amount of space in the bottom. So to even it out I made the container a flex and aligned it in the center.

Reply:
I don't think this is desireable though as the editor now jumps when you collapse / expand root element

Reply:
Wow thanks for noticing. It is even worse when the scroll appears. It will probably be best to leave it without an alignment then because even with padding it won't feel aligned when the scroll comes up.

Reply:
I think it's fine if it's slightly misaligned, yeah. I feel like with the editor it's more important that the editor itself is static than a slightly weird bottom padding of the container